scripts/deploy.bash

```diff
@@ -84,7 +84,8 @@ if ! [ "$JOBS_NODE" ]; then
     sudo kill -1 $pid
     sleep 0.5
     done
-
```

Comment:
is there another env var w3e should use to detect if were on a celery node?

Reply:
We should add one to indicate its a celery box, rather than checking for a file system.

Reply:
is that something you can do? im not sure the correct way to do so.. i just submitted this PR bc celery wasnt working in production the other day..